code/__DEFINES/traits.dm

```diff
@@ -235,6 +235,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_SIGN_LANG				"sign_language" //Galactic Common Sign Language
 #define TRAIT_NANITE_MONITORING	"nanite_monitoring" //The mob's nanites are sending a monitoring signal visible on diag HUD
 #define TRAIT_MARTIAL_ARTS_IMMUNE "martial_arts_immune" // nobody can use martial arts on this mob
+#define TRAIT_DUFFEL_CURSED "duffel_cursed" //You've been cursed with a living duffelbag, and can't have more added
```

Comment:
Is there a reason you decided to go with a trait marking a duffle curse, rather than just checking for an existing cursed dufflebag in the inventory of the person who's hit by the spell?

Reply:
Not particularly. What would your preferred method be?

Reply:
I feel this is maybe the simplest solution for now, so unless there was something making an inventory check more advantageous, maybe I'll keep it to the trait checks.